tutor/src/components/annotations/summary-toggle.jsx

```diff
@@ -38,12 +39,14 @@ export default class AnnotationSummaryToggle extends React.Component {
     if (!this.isViewable) { return null; }
 
     return (
-      <div className="annotation-summary-toggle">
-        <Icon
-          type="pencil-square-o"
-          onClick={User.annotations.ux.toggleSummary}
-        />
-      </div>
+      <TourAnchor id="student-highlighting-button">
+        <div className="annotation-summary-toggle">
+          <Icon
+            type="pencil-square-o"
+            onClick={User.annotations.ux.toggleSummary}
+          />
+        </div>
+      </TourAnchor>
     );
```

Comment:
You could let the `TourAnchor` act as the wrapper div by setting the `className` on it, such as:
```
  <TourAnchor id="student-highlighting-button" className="annotation-summary-toggle">
    <Icon type="pencil-square-o"  onClick={User.annotations.ux.toggleSummary} />
  </TourAnchor>
```